Corona-Warn-App/src/deviceForTesters/res/layout/fragment_test_keydownload.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:ignore="HardcodedText">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_margin="8dp"
+        android:orientation="vertical"
+        android:paddingBottom="32dp">
+
+        <LinearLayout
+            style="@style/card"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/spacing_tiny"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="8dp"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/info_metered_network"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Is metered network: ??" />
+
+            <Switch
+                android:id="@+id/allow_metered_toggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Hourly pkgs on metered networks" />
+
+            <LinearLayout
```

Comment:
Three layers of nested linear layouts!? I can only accept that for test fragment ;)